### DISCUSSION TITLE
[Question] How to disable/turn off/disconnect AC WiFi?

### DISCUSSION THREAD

**Discussion by techyporcupine** [omerfaruk-aran/esphome_samsung_hvac_bus#219] [Questions] [answered]
### Your Question

Hello! After using this project, I no longer need the built in WiFi capabilities of the Samsung AC unit, and I would like to turn them off. I have been unable to find a way to actually disable the networking on my unit. I have even tried a reset with POWER, SET, and MODE to have it just reconnect afterwards.

### Pre-Submission Checklist

- [X] I have searched for a similar question in open/closed issues.
- [X] I have checked the project’s documentation for relevant information.
- [X] I have provided as much detail as possible to help answer my question.

### HVAC System Type

NASA (Bus Protocol)

### Samsung HVAC Device Model

AR12TSFABWKNCV

### ESP Device Model

M5 Stamp

### Connection Pins

F1/F2

**Comment by omerfaruk-aran** [comment]
### Disabling Wi-Fi on Samsung AR12TSFABWKNCV Air Conditioner

If you'd like to disable the built-in Wi-Fi feature of your Samsung AR12TSFABWKNCV model air conditioner, you can try the following steps:

1. **Remove the Device from SmartThings:**
   - Open the SmartThings app on your smartphone.
   - Select your air conditioner and go into the device settings.
   - Tap on "Remove Device" to delete the air conditioner from the app.

2. **Reset the Air Conditioner's Wi-Fi Settings:**
   - Using the remote control, press and hold the "SET" and "MODE" buttons simultaneously for a few seconds.
   - When "AP" or a similar indicator appears on the display, the Wi-Fi settings should be reset.

After following these steps, the Wi-Fi connection of your air conditioner should be disabled. 

For more details, refer to the [Samsung Support Page](https://www.samsung.com/us/support/).

**Reply by techyporcupine** [reply]
1: there are no device settings to go into. There are "air conditioner options" but they do not have a way to remove the device.
2: this does not reset it.

**Reply by omerfaruk-aran** [reply]
https://youtu.be/-oKLzWGehGo?t=124

Can you try the step in the YouTube video in the link?

**Reply by techyporcupine** [reply]
That connects it, not disconnects it. Even after putting it in AP mode as shown it is still connected to WiFi

**Reply by omerfaruk-aran** [reply]
This is impossible. Wifi settings are reset when you switch to AP mode. As in the installation in the video, when AP is written on the screen, wait for a while, then AP will appear automatically. If it still does not reset, it would be useful to contact SAMSUNG customer service.

**Reply by techyporcupine** [reply]
Hmm j wonder if it still shows the WiFi symbol even if it isn't connected. I'll try checking my router to see if it disconnects next time.

**Reply by omerfaruk-aran** [reply]
Yes it can. Even if there is a wifi symbol, it is useful to check whether it is connected via the router.

**Reply by techyporcupine** [reply]
While AP mode is turned on, the device is no longer connected to WiFi, but after waiting for it to turn off it just reconnects.

**Reply by techyporcupine** [reply, marked as answer]
I have found a solution. It is not good by any means, but it works. 
In my Access Point management software, I was able to "Block" the device, which is blocking it from being on the WiFi and in turn, turned off the WiFi light on the display.

This did what I want but has a couple drawbacks. One of which is that the AC will still be trying to connect to the network and using up valuable 2.4GHz airtime.
